--- Makefile.in~	2000-09-05 09:32:11.000000000 +0200
+++ Makefile.in	2009-04-06 09:08:23.000000000 +0200
@@ -32,7 +32,7 @@
 	$(CC) $(LDFLAGS) -o enigma $(ENIGMA) $(LIBS)
 
 install: enigma
-	$(INSTALL_PROGRAM) enigma $(bindir)/enigma
+	$(INSTALL_PROGRAM) -D enigma $(bindir)/enigma
 	-chown root.games $(bindir)/enigma && chmod 2755 $(bindir)/enigma
 	#$(INSTALL_DATA) $(srcdir)/enigma.1 $(mandir)/man1/enigma.1
 	mkdir -p $(datadir)/enigma
